apps/gallery/js/ImageEditor.js

```diff
@@ -298,10 +300,16 @@ function editPhoto(n) {
     var imagesize = metadata.width * metadata.height;
     var maxsize = CONFIG_MAX_EDIT_PIXEL_SIZE || CONFIG_MAX_IMAGE_PIXEL_SIZE;
 
+    LazyLoader.load(['shared/js/media/jpeg-exif.js'], () => {
```

Comment:
The file might not be a jpeg. You should only do this if it is jpeg, I think.
Also, you seem to be speculatively loading the exif with the assumption that it will be read before the user finished editing the image. That is a good idea, but I think you should add a comment making it clear that that is what you're doing.